apps/web/pages/success.tsx

```diff
@@ -57,6 +54,26 @@ export default function Success(props: inferSSRProps<typeof getServerSideProps>)
   };
 
   const eventName = getEventName(eventNameObject);
+  const needsConfirmation = eventType.requiresConfirmation && reschedule != "true";
+
+  useEffect(() => {
+    const users = eventType.users;
+    // TODO: We should probably make it consistent with Webhook payload. Some data is not available here, as and when requirement comes we can add
+    sdkActionManager!.fire("bookingSuccessful", {
+      eventType,
+      date: date.toString(),
+      duration: eventType.length,
+      organizer: {
+        name: users[0].name || "Nameless",
+        email: users[0].email || "Email-less",
```

Comment:
Can events really be both Nameless and Email-less, who do they belong to then?

Reply:
Nameless - certainly; it is when a user hasn't got a username configured. I don't believe email is nullable though; but if it is we may want to change that.